manage.py

```diff
@@ -1,7 +1,9 @@
 from __future__ import print_function
-
+from flask_script import Server
 import os
 
+SERVER_NAME = __import__('config').Config.SERVER_NAME
```

Comment:
You don't need to do this. Instead, you can use `app.config['SERVER_NAME']`

Reply:
@SaptakS I have tried it, but not able to import it. Then I have used this.

Reply:
You don't need to import. You are already importing app. And the configurations are already present in the flask app. So you can directly do something like `host = app.config['SERVER_NAME'].split(":")[0]` and `port = app.config['SERVER_NAME'].split(":")[1]`

Reply:
Got it @SaptakS